libengine/dpengine/tile.h

```diff
@@ -68,8 +68,13 @@ DP_INLINE int DP_tile_count_round(int i)
 
 DP_INLINE DP_TileCounts DP_tile_counts_round(int width, int height)
 {
+#ifdef __cplusplus
```

Comment:
Other files solve this like:
```c
DP_Struct s = {...};
return s;
```
Because that's shorter than an ifdef and works in both C and C++.